docs/create-and-manage-a-shop.md

```diff
@@ -0,0 +1,101 @@
+
+## Create a shop
+
+Once you've created your first user, you're prompted to create a shop. Enter the name of the shop in the text field and click **Create Shop**. The first shop you create is automatically assigned to the primary shop type. Each Open Commerce installation can have only one *primary shop*. Any subsequent shops you may create is assigned a *merchant shop* type.
+
+You can create multiple shops in Open Commerce. This is referred to as *multitenancy*. To create additional shops, click your shop name in the upper left corner of the screen and choose **New Shop** from the drop-down menu. Enter the name of the new shop and click **Create Shop**. This new shop is automatically assigned to the merchant shop type.
+![Create second shop](_assets/02-admin-create-shop-2.png)
+
+You can switch between shops using the admin portal. Click the name of the current shop in the upper left corner and then choose the shop you want to switch to.
+![Switch shop](_assets/02-admin-switch-shop.png)
+
+Once you have created a user and set up a shop, the admin portal will look like this.
+![Admin](_assets/02-admin-user-shop-setup.png)
+
+## Accounts
+
+You'll find account management and settings options by clicking Accounts in the side menu of the Open Commerce Admin portal.
+
+Within Accounts you can access the following options::
+- Staff Members: View and modify the accounts involved with shop management and the different permission groups to which they belong. The first account created is populated under Owner. You can also create additional groups and invite users here.
+- Customers: View customers of the shop..
+- Invitations: View the outstanding sent invitations. Once the user accepts an invitation it is no longer shown here.
+
+### Create Group
+
+To create a new group, click Create Group under Staff Members, and add the following information in the pop-up modal:
+- Name: Group name. Required.
+- Slug: A URL-friendly slug. Optional. If you don't provide this, the group name is used as the slug.
+- Description: A brief description of the group. Optional.
+- Roles: Select the appropriate permissions for the group from the dropdown menu. You can create, modify and delete permissions for different features of the app here.
+
+Don't forget to save your changes!
+
+### Invite Staff Member
+
+You can invite people to be a part of your shop management by clicking Invite Staff Member. You'll need to enter their Name and Email ID, and then you can select the groups to which you want to add this member.
+
+Once you've successfully sent the invitation, the recipient should receive an email.
+
+> Note: You'll need to set up an email service to send out invitations. See [Email Configuration](75-email-configuration.md) for more information.
+
+## Payments
+
+To set up Payments, go to Settings > Payments in the admin dashboard. Click to enable or disable a payment method. Enabled payment methods are visible to the customer at checkout.
```

Comment:
Settings > payments should have the same bold format as similar instructions below